sites/all/themes/springboard/less/overrides.less

```diff
@@ -1004,9 +1014,9 @@ label.control-label input:disabled ~ .control__indicator:after {
   margin-right: 15px !important;
 }
 
-h1.getsupport-title {
-  padding-left: 15px;
-}
+//h1.getsupport-title {
```

Comment:
Instead of commenting it out, please remove this code block if its not required.